### PR TITLE
adding iOS binding

### PR DESCRIPTION
**Pull request by matzew**

### CODE REVIEW

File: apb.yml [hunk 1]
```diff
@@ -33,12 +33,22 @@ plans:
...
       title: Your Google Key for Firebase Cloud Messaging
       type: string
+      display_group: Android
     - name: projectNumber
-      required: True
       title: Your FCM Project Number, needed on the mobile client for connecting with FCM
       type: string
+      display_group: Android
+    - name: cert
+      title: iOS .p12 file
```
Comment (pb82) [inline apb.yml]: should we say that this needs to be base64 encoded?

Comment (matzew) [reply apb.yml]: make a suggestion

Comment (pb82) [reply apb.yml]: What about: `iOS .p12 file (encode contents in base64 before pasting)`?

File: apb.yml [hunk 2]
```diff
@@ -33,12 +33,22 @@ plans:
...
+      type: string
+      display_type: textarea
+      display_group: ios
```
Comment (pb82) [inline apb.yml]: Maybe call the display group `iOS` (upcase) because this is also used as the section caption

**Review by pb82 — APPROVED** [review]
👍

**Pull request merged**